migration/v2.20.0-ck8s4/upgrade-cluster.md

```diff
@@ -0,0 +1,34 @@
+# Upgrade v2.20.0-ck8s3 to v2.20.0-ck8s4
+
+## Prerequisites
+
+As this patch version only includes config changes that are not applied to the cluster, there are no prerequisites.
+
+## Steps that can be done before the upgrade - non-disruptive
+
+1. Checkout the new release: `git switch -d v2.20.0-ck8s4`
+
+1. Switch to the correct remote: `git submodule sync`
+
+1. Update the kubespray submodule: `git submodule update --init --recursive`
+
+1. Run the following to create the new config file for both sc and wc.
+
+    ```console
+    export CK8S_KUBESPRAY_PATH=/path/to/compliantkubernetes-kubespray
+
+    for cluster in "sc" "wc"; do
+        cp "${CK8S_KUBESPRAY_PATH}"/config/common/group_vars/all/ck8s-kubespray-general.yaml \
+        "${CK8S_CONFIG_PATH}"/"${cluster}"-config/group_vars/all/ck8s-kubespray-general.yaml
+    done
+    ```
```

Comment:
This step should be added to more or another migration doc (`v2.21`?), as I don't think we will upgrade most clusters to this patch version, and the prepare scripts don't work if the file isn't created.

Reply:
I was planning on migrating all environments to this patch version, don't need to do it during a maintenance window since the only difference is the addition of this file. But regardless of that I agree that this migration step should be added to the migration docs for `v2.21`, because we shouldn't have `v2.20.0-ck8s4` as a strictly required part of the upgrade path